main.py

```
@@ -1,21 +1,39 @@
-from fastapi import FastAPI
+import sentry_sdk
+import os
+import torch
+import re
+import json
+
+from fastapi import FastAPI, Request
 from pydantic import BaseModel
 from typing import List
 from utils.model import generate_simulation_result, detect_profanity
```

Ruff (0.11.9)

`utils.model.generate_simulation_result` imported but unused

Remove unused import: `utils.model.generate_simulation_result`

(F401)